### DISCUSSION TITLE
Pages Deployment fails with error code 200, builds successfully

### DISCUSSION THREAD

**Discussion by alienfunbug** [community/community#39907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm attempting to npm deploy a react app to github pages. I've made many deployments in the past but all of a sudden I am receiving error code 200 on deployment step.

I've tried deleting all jobs and redeploying, as well as deploying manually. The issue is still occurring. Here's the output from the github deployment:

> Warning: Input 'emit_telemetry' has been deprecated with message: Use of this input causes the action to do nothing. You should remove this build step from your workflow.
> Run actions/deploy-pages@v1
> with:
> emit_telemetry: false
> token: ***
> timeout: 600000
> error_count: 10
> reporting_interval: 5000
> artifact_name: github-pages
> preview: false
> Actor: github-pages[bot]
> Action ID: 3537811468
> Artifact URL: https://pipelines.actions.githubusercontent.com/md6wek8JfMAL6JcQXj6SoBBT2dfRoUaLHatYRmRfYJp95TeIVb/_apis/pipelines/workflows/3537811468/artifacts?api-version=6.0-preview
> {"count":1,"value":[{"containerId":10763181,"size":472012800,"signedContent":null,"fileContainerResourceUrl":"https://pipelines.actions.githubusercontent.com/md6wek8JfMAL6JcQXj6SoBBT2dfRoUaLHatYRmRfYJp95TeIVb/_apis/resources/Containers/10763181","type":"actions_storage","name":"github-pages","url":"https://pipelines.actions.githubusercontent.com/md6wek8JfMAL6JcQXj6SoBBT2dfRoUaLHatYRmRfYJp95TeIVb/_apis/pipelines/1/runs/117/artifacts?artifactName=github-pages","expiresOn":"2022-11-25T04:51:42.9487824Z","items":null}]}
> Creating deployment with payload:
> {
> "artifact_url": "https://pipelines.actions.githubusercontent.com/md6wek8JfMAL6JcQXj6SoBBT2dfRoUaLHatYRmRfYJp95TeIVb/_apis/pipelines/1/runs/117/artifacts?artifactName=github-pages&%24expand=SignedContent",
> "pages_build_version": "28be79d5c86a822c8d5f50a352c4d32966e15a00",
> "oidc_token": "***"
> }
> Created deployment for 28be79d5c86a822c8d5f50a352c4d32966e15a00
> {"page_url":"[https://contactfail.com/","status_url":"https://api.github.com/repos/alienfunbug/ketrio/pages/deployment/status/28be79d5c86a822c8d5f50a352c4d32966e15a00","preview_url":""}](https://contactfail.com/%22,%22status_url%22:%22https://api.github.com/repos/alienfunbug/ketrio/pages/deployment/status/28be79d5c86a822c8d5f50a352c4d32966e15a00%22,%22preview_url%22:%22%22%7D)
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Unable to get deployment status.
> Too many errors, aborting!
> Error: Failed with status code: 200
> Deployment cancelled with https://api.github.com/repos/alienfunbug/ketrio/pages/deployment/cancel/28be79d5c86a822c8d5f50a352c4d32966e15a00

I've also tried deploying a push that was successfully deployed last week, and now that one fails. 
Please let me know if theres anything wrong on my end!

**Comment by davay42** [comment]
Same error here! Seems to happen suddenly and persists. Any ideas? @alienfunbug

**Comment by DSDmark** [comment]
Hi @alienfunbug,

can you provide your repo link and check this out:- https://github.com/community/community/discussions/38632#discussioncomment-4092967

**Comment by davay42** [comment]
@DSDmark I've tried to delete and recreate the environment as suggested in that post. But the error still persists. Here's the link https://github.com/trika-yoga/trikayoga.ru/actions

<img width="1241" alt="Screenshot 2022-11-24 at 22 26 35" src="https://user-images.githubusercontent.com/6184449/203819840-a477bd18-0003-4869-a525-22e8834a56be.png">


<img width="1283" alt="Screenshot 2022-11-24 at 21 29 53" src="https://user-images.githubusercontent.com/6184449/203819556-da2002af-f0b7-49db-ba0d-12f845bb0bf4.png">

<img width="1283" alt="Screenshot 2022-11-24 at 21 37 56" src="https://user-images.githubusercontent.com/6184449/203819636-f7eb1faf-34d3-406f-83a0-33a971f51b89.png">


Here's the output of the `deploy` job of the Pages action. 
```
Warning: Input 'emit_telemetry' has been deprecated with message: Use of this input causes the action to do nothing. You should remove this build step from your workflow.
Run actions/deploy-pages@v1
Actor: github-pages[bot]
Action ID: 3541805104
Artifact URL: https://pipelines.actions.githubusercontent.com/RY5TCK83xuNYOBQTzHgFK4PqSgnrjlEdlo3ZrsND7CtmfdKi1X/_apis/pipelines/workflows/3541805104/artifacts?api-version=6.0-preview
{"count":1,"value":[{"containerId":13222335,"size":683417600,"signedContent":null,"fileContainerResourceUrl":"https://pipelines.actions.githubusercontent.com/RY5TCK83xuNYOBQTzHgFK4PqSgnrjlEdlo3ZrsND7CtmfdKi1X/_apis/resources/Containers/13222335","type":"actions_storage","name":"github-pages","url":"https://pipelines.actions.githubusercontent.com/RY5TCK83xuNYOBQTzHgFK4PqSgnrjlEdlo3ZrsND7CtmfdKi1X/_apis/pipelines/1/runs/34/artifacts?artifactName=github-pages","expiresOn":"2022-11-25T15:16:27.6861401Z","items":null}]}
Creating deployment with payload:
{
	"artifact_url": "https://pipelines.actions.githubusercontent.com/RY5TCK83xuNYOBQTzHgFK4PqSgnrjlEdlo3ZrsND7CtmfdKi1X/_apis/pipelines/1/runs/34/artifacts?artifactName=github-pages&%24expand=SignedContent",
	"pages_build_version": "7e44ac09a7869bec9824752c65131f31ae4db77e",
	"oidc_token": "***"
}
Created deployment for 7e44ac09a7869bec9824752c65131f31ae4db77e
{"page_url":"https://trikayoga.ru/","status_url":"https://api.github.com/repos/trika-yoga/trikayoga.ru/pages/deployment/status/7e44ac09a7869bec9824752c65131f31ae4db77e","preview_url":""}
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Unable to get deployment status.
Too many errors, aborting!
Error: Failed with status code: 200
Deployment cancelled with https://api.github.com/repos/trika-yoga/trikayoga.ru/pages/deployment/cancel/7e44ac09a7869bec9824752c65131f31ae4db77e

```

**Comment by prcr** [comment]
The same thing is also happening on this repository:

https://github.com/codacy/docs/actions/runs/3539847809

Most probably it's an issue on the GitHub side, but everything's green on their [status page](https://www.githubstatus.com/).

**Comment by ArnaBlack** [comment]
hi, have the same error
<img width="1769" alt="Снимок экрана 2022-11-24 в 19 00 10" src="https://user-images.githubusercontent.com/48154833/203826299-507f7c0e-9652-4611-a56d-f72421d0a9d8.png">

**Comment by yoannchaudet** [comment]
👋 We let a regression slipped that broke deployment status reporting for larger sites. Most of these sites were being deployed successfully but I believe the larger one were not. It should be fixed now, the team will look at addressing the alerting gap that prevented to catch that earlier and backfill an availability incident when they are back in the office Monday (🦃 break for the team in the US).

If you are still impacted let us know here, I'll be keeping an eye.

**Reply by alienfunbug** [reply]
Thanks so much for the hard work during the holiday! I am good to go on this end as well! Appreciated!

**Reply by autumngreenbean** [reply]
Error has gone away for my pages deploy, however changes pushed to repository do not seem to update or appear. I have made a push earlier this morning with a successful deploy, however there seems to be no apparent changes.

**Comment by davay42** [comment]
Yes, the deploys are successful now! Thanks!

**Comment by Parking-Master** [comment]
YES! It works!

**Comment by Parking-Master** [comment]
Thanks to the team for fixing it so fast (on thanksgiving)

**Comment by autumngreenbean** [comment]
Page successfully deployed after having the same error 200. Thanks for clearing this up!

**Reply by renatocoutinho** [reply]
That error went away, but now it keeps timing out (my project is huge but it did finish before).

**Reply by yoannchaudet** [reply]
Link to repo?

**Reply by renatocoutinho** [reply]
github.com/covid19br/covid19br.github.io

**Reply by yoannchaudet** [reply]
The [last deployment for your site went through](https://github.com/covid19br/covid19br.github.io/actions/runs/3537531063) but I really recommend you make it lose weight.

At 7.2GB you are 7 times over our [documented limit](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits). We have a hard limit on how long a deployment can take and you are close to it.

If you look at [deployment trends](https://github.com/covid19br/covid19br.github.io/actions) for this site you will see a lot of cancelled deployments. These are the result of very fast commits in a row (by default when a new commit comes in we cancel the current deployment to let the last commit takeover).

You could benefit using a [custom workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow) to have more control on this default logic.

**Reply by autumngreenbean** [reply]
The error has gone away and the deploy is successful, but none of my changes that I have pushed seem to be appearing on my Pages website.

**Comment by sveta-slepner** [comment]
We are still experiencing issues with deploying Github pages. It seems that tasks simply time out (there is no error) and even using `continue-on-error: true` doesn't help it to move on to the next steps...
We were using `peaceiris/actions-gh-pages@3` but similar actions fail in the same manner.

**Reply by yoannchaudet** [reply]
If it times out you may be deploying a large site. Do you have a link to share or can you check the size of your artifacts?

**Reply by sveta-slepner** [reply]
I can't unfortunately, but we're simply deploying allure reports which indeed upload a large number of files. But it worked fine before, with the exact same amount, and fails even when running with a smaller scope which generates a smaller deploy.

**Reply by yoannchaudet** [reply]
If you are within the [service limits](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits), please [open a support ticket](https://support.github.com/contact) and we will assist you!

**Comment by j-makwana** [comment]
Experiencing the same now. Its a fairly small project but it's just stuck on deployment.
![Uploading Screenshot 2024-01-17 at 12.28.08 AM.png…]()
